README.md

```diff
@@ -121,19 +121,15 @@ scrape_configs:
       module: [http_2xx]  # Look for a HTTP 200 response.
     static_configs:
       - targets:
-        - prometheus.io   # Target to probe
+        - http://prometheus.io    # Target to probe with http.
+        - https://prometheus.io   # Target to probe with https.
+        - http://example.com:8080 # Target to probe with http on port 8080.
     relabel_configs:
       - source_labels: [__address__]
-        regex: (.*)(:80)?
```

Comment:
This is only to optionally strip `:80` from the target.. for example `http://example.com:80` would be come `http://example.com`.  This is unnecessary for an example config with the full URLs as targets.